cloud/defaults.go

```diff
@@ -202,6 +207,22 @@ func GetDefaultUbuntuImage(k8sVersion string) (*infrav1.Image, error) {
 	return defaultImage, nil
 }
 
+// GetDefaultWindowsImage returns the default image spec for Windows.
+func GetDefaultWindowsImage(subID string) *infrav1.Image {
+	// Temporarily hardcoded to shared image gallery while publishing is done in Market Place
+	defaultImage := &infrav1.Image{
+		SharedGallery: &infrav1.AzureSharedGalleryImage{
```

Comment:
The Marketplace Gallery Image is being published. As soon as it is I will update this section with the Market Place info

Reply:
The image is now published: `az vm image list --publisher cncf-upstream --offer capi-windows -o table --all`